File: ELEVSTRS.java
import java.util.*;
import java.lang.*;
import java.io.*;

class ELEVSTRS
{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int t = scanner.nextInt();
        while(t>0) 
        {
            t--;
            double n = scanner.nextDouble();
            double v1 = scanner.nextDouble();
            double v2 = scanner.nextDouble();
            v1 *= v1 * 2;
            v2 *= v2;
            String toPrint = "";
            if(v1 < v2) 
                toPrint = "Elevator";
            else
                toPrint = "Stairs";
            System.out.println(toPrint); 
        }
    }
}

File: ELEVSTRS.py
t = int(input())
while t>0:
    t -= 1
    n, v1, v2 = [int(num) for num in input().split()]
    v1 *= v1 * 2
    v2 *= v2
    if v1 < v2:
        print("Elevator")
    else:
        print("Stairs")